### PR TITLE
changes readme.md to say four maps

### PR DESCRIPTION
**Pull request by boodaliboo**
we have 4 maps listed
#### Changelog

:cl:   
tweak: put the right ammount of maps
/:cl:

### CODE REVIEW

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Yogstation currently comes equipped with the following maps.

Im getting tired of this being a mistake

**Comment by patpol4** [comment]
Are you sure

**Review by Ktlwjec1 — CHANGES_REQUESTED** [review]
Five is correct, it's the map names+links that aren't:
```
* [BoxStation (default)](https://wiki.yogstation.net/wiki/BoxStation)
* [DeltaStation](https://wiki.yogstation.net/wiki/DeltaStation)
* [EclipseStation](https://wiki.yogstation.net/wiki/Maps)
* [MetaStation](https://wiki.yogstation.net/wiki/MetaStation)
* [OmegaStation](https://wiki.yogstation.net/wiki/OmegaStation)
```

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ https://github.com/tgstation/tgstation-server
 
 ## MAPS
 
-Yogstation currently comes equipped with five maps.
+Yogstation currently comes equipped with four maps.
```
Comment (alexkar598) [inline README.md]:
```suggestion
Yogstation currently comes equipped with the following maps.
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**